src/sections/safe.tsx

```diff
@@ -201,6 +245,14 @@ function SafeSection() {
           ) : null}
         </div>
       ))}
+      <h2>approve hopr token to hopr channels</h2>
+      <span>allowance: {formatEther(BigInt(allowanceData?.toString() ?? '0'))}</span>
+      <button
+        disabled={!safe.selectedSafeAddress || !signer}
+        onClick={proposeApproveTransactionToSafe}
```

Comment:
The function that we are calling - please move it to Redux and make it accept data it needs as payload